components/provider/src/structs/icu4x.rs

```diff
@@ -0,0 +1,28 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
Should this be a separate `example` or at least behind some flag?

Reply:
I'll move this into the hello_world module when I fix #415